index.src.html

```diff
@@ -1879,6 +1942,15 @@
   <dd>
     Configures latency related behaviors for this codec. See {{LatencyMode}}.
   </dd>
+
+  <dt><dfn dict-member for=VideoEncoderConfig>colorSpace</dfn></dt>
+  <dd>
+    Configures the {{VideoColorSpace}} to be used in the output
+    {{VideoDecoderConfig}}. May also be used to set in-band color information
+    while encoding. If not provided, the codec will attempt to use the
+    {{VideoFrame}}.{{VideoFrame/colorSpace}} for the first frame passed to
+    {{VideoEncoder/encode()}} following this {{VideoEncoderConfig}}.
```

Comment:
What happens if this attempt fail ?

Reply:
Nothing in particular, following the steps the the output `colorSpace` value will be `null`.